pages/about.vue

```diff
@@ -0,0 +1,173 @@
+      the DAEDALUS model and DAEDALUS Explore dashboard.
+    </p>
+    <h4>Disclaimer and Licensing</h4>
+    <h5>Model code license</h5>
+    <p>
+      The DAEDALUS model code is provided “as is”, without warranty of any kind,
+      express or implied, including but not limited to the warranties of
+      merchantability, fitness for a particular purpose and noninfringement.
+    </p>
+    <p>
+      In no event shall the authors or copyright holders be liable for any
+      claim, damages or other liability, whether in an action of contract, tort
+      or otherwise, arising from, out of or in connection with the software or
+      the use or other dealings in the software.
+    </p>
+    <h5>Dashboard Icon licenses</h5>
+    <p>
+      <strong>rotate-device.svg</strong><br>
+      Icons by
+      <a
+        href="https://github.com/Esri/calcite-ui-icons?ref=svgrepo.com"
+        target="_blank"
+      >Esri</a>
+      in MIT License via
+      <a href="https://www.svgrepo.com/" target="_blank">SVG Repo</a>
+    </p>
+
+    <p>
+      <strong>carbonSEEarth.svg</strong><br>
+      Icons by
+      <a
+        href="https://github.com/carbon-design-system/carbon?ref=svgrepo.com"
+        target="_blank"
+      >Carbon Design</a>
+      in Apache License via
+      <a href="https://www.svgrepo.com/" target="_blank">SVG Repo</a>
+    </p>
+
+    <p>
+      <strong>vectopusVaccine.svg</strong><br>
+      Vectors and icons by
+      <a
+        href="https://github.com/Vectopus/Atlas-icons-font?ref=svgrepo.com"
+        target="_blank"
+      >Vectopus</a>
+      in MIT License via
+      <a href="https://www.svgrepo.com/" target="_blank">SVG Repo</a>
+    </p>
+
+    <p>
+      <strong>pictogrammersMaterialBacteria.svg</strong><br>
+      Icons by
+      <a
+        href="https://github.com/carbon-design-system/carbon?ref=svgrepo.com"
+        target="_blank"
+      >Carbon Design</a>
+      in Apache License via
+      <a href="https://www.svgrepo.com/" target="_blank">SVG Repo</a>
+    </p>
+    <h3>References</h3>
+    <p>
+      Haw, David J., Giovanni Forchini, Patrick Doohan, Paula Christen, Matteo
+      Pianella, Robert Johnson, Sumali Bajaj, et al. 2022. “Optimizing Social
+      and Economic Activity While Containing SARS-CoV-2 Transmission Using
+      DAEDALUS.” Nature Computational Science 2 (4): 223–33.
+    </p>
+    <div class="d-flex flex-wrap justify-content-between">
+      <img
+        class="p-2"
+        style="width: 200px"
+        title="Imperial Logo"
+        src="~/assets/img/imperial_logo.svg"
+        alt="Imperial Logo"
+      >
+      <img
+        class="p-2"
+        title="Prepare logo"
+        style="width: 300px"
+        src="~/assets/img/prepare-logo.png"
+        alt="Prepare logo"
+      >
+      <img
+        class="p-2"
+        title="Jameel logo"
+        style="width: 200px"
+        src="~/assets/img/jameel-logo.jpg"
+        alt="Jameel logo"
+      >
+    </div>
+  </CContainer>
```

Comment:
I think we need more space below the images, which currently seem to be basically flush with the bottom of the page.

Reply:
Also consider vertical image layout: https://www.arbomap.org/dengue/about

Reply:
Just noticed arbomap seems to have a png Jameel logo, that could be perfect
https://github.com/mrc-ide/arbomap/blob/main/src/assets/ICL_Jameel_Full-Lock-up_RGB%20Transparent.png

Reply:
> Also consider vertical image layout: https://www.arbomap.org/dengue/about
 Is this what we want as this becomes and hirearchy? ellie said left to right?

Reply:
That’s a good point and I hadn’t noticed Ellie specified that.